.gitignore

```diff
@@ -4,3 +4,4 @@ test/data/
 test/assets/module/*/config/module.config.php
 clover.xml
 coveralls-upload.json
+docker-compose.yml
```

Comment:
We shouldn't ignore that file, but `docker-compose.override.yml`.

Reply:
I've changed this around to override.